### PR TITLE
New KS scorer

### PR DESCRIPTION
**Pull request by us8945**
The scorer was created as per request from customer.

### CODE REVIEW

File: scorers/classification/binary/kolmogorov_smirnov.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+
+        :return: score
+        """
+
+        try:
+            """Initialize logger to print additional info in case of invalid inputs(exception is raised) and to enable debug prints"""
+            logger = self.logger
+            from h2oaicore.systemutils import loggerinfo
+            # loggerinfo(logger, "Start Gamma Deviance Scorer.......")
```
Comment (arnocandel) [inline scorers/classification/binary/kolmogorov_smirnov.py:70]: Why those comments?

Comment (us8945) [reply scorers/classification/binary/kolmogorov_smirnov.py]: Extra comments removed.

File: scorers/classification/binary/kolmogorov_smirnov.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        CustomScorer.__init__(self)
+
+    @staticmethod
+    def do_acceptance_test():
+        """
+        Whether to enable acceptance tests during upload of recipe and during start of Driverless AI.
+        Acceptance tests perform a number of sanity checks on small data, and attempt to provide helpful instructions
+        for how to fix any potential issues. Disable if your recipe requires specific data or won't work on random data.
+        """
+        return False
```
Comment (arnocandel) [inline scorers/classification/binary/kolmogorov_smirnov.py:42]: why disabled? isn't it generic enough to work?

Comment (us8945) [reply scorers/classification/binary/kolmogorov_smirnov.py:42]: It is generic scorer, should work for all cases. Will make a change and push

**Pull request merged**